x/distribution/types/fee_pool.go

```diff
@@ -0,0 +1,50 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// total accumulation tracker
+type TotalAccum struct {
+	UpdateHeight int64   `json:"update_height"`
+	Accum        sdk.Dec `json:"accum"`
+}
+
+func NewTotalAccum(height int64) TotalAccum {
+	return TotalAccum{
+		UpdateHeight: height,
+		Accum:        sdk.ZeroDec(),
+	}
+}
+
+// update total validator accumulation factor
+func (ta TotalAccum) Update(height int64, accumCreatedPerBlock sdk.Dec) TotalAccum {
+	blocks := height - ta.UpdateHeight
+	ta.Accum = ta.Accum.Add(accumCreatedPerBlock.Mul(sdk.NewDec(blocks)))
```

Comment:
I wonder if we should add a `MulInt` function to `sdk.Dec`, might be more efficient

Reply:
yeah, I think you're correct - adding now

Reply:
lol - it already existed... okay updating to use it!

Reply:
I'm happy `MulInt` is getting used :D 
I was worried I'd be accused of micro-optimizing for making it lol